lib/notifications/display.dart

```diff
@@ -25,6 +25,10 @@ AndroidNotificationHostApi get _androidHost => ZulipBinding.instance.androidNoti
 
 /// Service for configuring our Android "notification channel".
 class NotificationChannelManager {
+  /// The channel ID we use for our one notification channel, which we use for
+  /// all notifications.
+  // TODO(launch) check this doesn't match zulip-mobile's current or previous
+  // channel IDs
```

Comment:
nit: in a TODO comment (or other labeled comment), indent any continuation lines to keep them grouped under the heading:
```suggestion
  // TODO(launch) check this doesn't match zulip-mobile's current or previous
  //   channel IDs
```